Documentation/kubernetes-on-baremetal.md

```diff
@@ -1,13 +1,34 @@
 # Kubernetes Installation on Bare Metal &amp; CoreOS
 
-This guide walks a deployer through launching a multi-node Kubernetes cluster on bare metal servers running CoreOS.
-After completing this guide, a deployer will be able to interact with the Kubernetes API from their workstation using the kubectl CLI tool.
+This guide walks a deployer through launching a multi-node Kubernetes cluster on bare metal servers running CoreOS. After completing this guide, a deployer will be able to interact with the Kubernetes API from their workstation using the kubectl CLI tool.
 
-## Deployment Prerequisites
+## Deployment Requirements
 
-### CoreOS Installation
+### CoreOS Version
 
-For all nodes running Kubernetes components (masters & workers), you must use a CoreOS version 962.0.0+ for the `kubelet-wrapper` script to be present in the image. If you wish to use an earlier version (e.g. from the 'stable' channel) see [kubelet-wrapper](kubelet-wrapper.md) for more information.
+All Kubernetes controllers and nodes must use a CoreOS version 962.0.0+ for the `kubelet-wrapper` script to be present in the image. If you wish to use an earlier version (e.g. from the 'stable' channel) see [kubelet-wrapper](kubelet-wrapper.md) for more information.
+
+### Kubernetes Pod Network
+
+These guides use [flannel][coreos-flannel] as a software-defined overlay network to manage routing of the [pod network][pod-network]. However, bare metal is a common platform where a self-managed network may be used, due to the flexibility provided by physical networking gear. One common alternative is to use [Calico][calico-networking] networking, avoiding the use of an overlay and allowing interop with physical networking gear using BGP.
```

Comment:
This configuration uses the [flannel] overlay network to manage the [pod network]. Many bare metal configurations may instead have an existing self-managed network. In this scenario, it is common to use [Calico] to manage pod network policy while omitting the overlay network, and interoperating with existing physical network gear over BGP.